README.md

```diff
@@ -41,6 +41,9 @@ FHIR Works on AWS is powered by single-function components. These functions prov
 
 - [Interface](https://github.com/awslabs/fhir-works-on-aws-interface) - Defines communication between the components.
 - [Routing](https://github.com/awslabs/fhir-works-on-aws-routing) - Accepts HTTP FHIR requests, routes it to the other components, logs the errors, transforms output to HTTP responses and generates the [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
+  - What is the recommended transport layer security (TLS) setting? 
+  
+    FHIR Works on AWS does not deploy a custom domain, so API Gateway does not allow FHIR Works on AWS to require TLS 1.2. Customers should configure FHIR Works on AWS to meet their internal security policies.
```

Comment:
`to meet their internal security policies.` -> we should be more descriptive. What is AWS's advice? perhaps link to an AWS page

We should say something closer to "we advise x,y,z to secure your transport layer"

Reply:
https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-custom-domain-tls-version.html